datahub-web-react/src/alchemy-components/components/Select/Nested/NestedSelect.tsx

```diff
@@ -60,7 +60,7 @@ export const selectDefaults: SelectProps = {
     isDisabled: false,
     isReadOnly: false,
     isRequired: false,
-    isMultiSelect: false,
+    isMultiSelect: true,
```

Comment:
hm do we want to change the default here? that could affect other places using nested select obviously

Reply:
Right so before this change _everything_ was multi select. I could not find a case where this actually worked at all. 

So technically everything should have been single select but it was not. I am double confirming now..

Reply:
Places where this component is used today:

- Entity Type Filter in Search
- Forms - DomainsSelector, GlossaryTermSelector, OwnershipSelector, OwnershipTypeSelector 
- Assertions List, Incident List Filters 

Going to make sure each was multi select by default currently.